org.openair.podcast.yml

```diff
@@ -0,0 +1,113 @@
+app-id: org.openair.podcast
+runtime: org.gnome.Platform
+runtime-version: "49"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm20
+command: OpenAir
+
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-data/openair:create
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --talk-name=org.freedesktop.Notifications
+
+modules:
+  - name: openair
+    buildsystem: simple
+    build-options:
+      build-args:
+        - --share=network
+      arch:
+        x86_64:
+          env:
+            BUNDLE_PATH: build/linux/x64/release/bundle
+        aarch64:
+          env:
+            BUNDLE_PATH: build/linux/arm64/release/bundle
+      append-path: /usr/lib/sdk/llvm20/bin:/run/build/openair/flutter/bin
+      prepend-ld-library-path: /usr/lib/sdk/llvm20/lib
+      env:
+        PUB_CACHE: /run/build/openair/.pub-cache
+        CXXFLAGS: -Wno-error=deprecated-literal-operator
+
+    build-commands:
+      - flutter pub get
+      - flutter build linux --release
+
+      - mkdir -p /app/openair
+      - cp -r $BUNDLE_PATH/* /app/openair/
+      - mkdir -p /app/bin
+
+      - |
+        cat > /app/bin/OpenAir << 'EOF'
+        #!/bin/bash
+        cd /app/openair
+        exec /app/openair/OpenAir "$@"
+        EOF
+
+      - chmod +x /app/bin/OpenAir
+
+      - |
+        cat > org.openair.podcast.desktop << 'EOF'
```

Comment:
same for desktop file (should be upstream) and appstream file below (should be upstream)